cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -98,43 +117,101 @@ started correctly.
-     > docker-machine stop cdap
+- Use the CDAP CLI in its own container (*cdap-cli*), against a remote CDAP instance at ``${CDAP_HOST}``::
+
+    $ docker run -it --name cdap-cli --rm caskdata/cdap-standalone cdap-cli.sh -u http://${CDAP_HOST}:11011
+
+- Use the CDAP CLI in its own container (*cdap-cli*), against the above *cdap-standalone* container using container linking::
+
+    $ docker run -it --link cdap-sdk:sdk --name cdap-cli --rm caskdata/cdap-standalone sh -c 'exec cdap-cli.sh -u http://${SDK_PORT_11011_TCP_ADDR}:${SDK_PORT_11011_TCP_PORT}'
+
+- Starting the Standalone CDAP, in the foreground, with ports forwarded::
+
+    $ docker run -it -p 11015:11015 -p 11011:11011 --name cdap-sdk caskdata/cdap-standalone cdap.sh start
+    
+- Starting the Standalone CDAP, in the foreground, with ports forwarded, and with debugging enabled::
+
+    $ docker run -it -p 11015:11015 -p 11011:11011 --name cdap-sdk caskdata/cdap-standalone cdap.sh start --enable-debug
+
+- For information on mounting volumes and sharing data with the container, see the
+  examples in Docker's documentation on `data volumes
+  <https://docs.docker.com/engine/tutorials/dockervolumes/#/adding-a-data-volume>`__.
+  You can either let Docker manage the storage in the container (the easiest) or you can 
+  `mount a host directory as a data volume
+  <https://docs.docker.com/engine/tutorials/dockervolumes/#/mount-a-host-directory-as-a-data-volume>`.
+  However, if you mount a host directory, you must make sure that it exists and that
+  permissions are set correctly. You pass such a directory using ``-v
+  /my/own/datadir:/opt/my/own/datadir``, where mounts the ``/my/own/datadir`` from the
+  host system as ``/opt/cdap/sdk/my/own/datadir`` in the container.
+
+Controlling the CDAP Instance
+-----------------------------
+
+- To control the CDAP instance, use this command, substituting one of ``start``, ``restart``, ``stop``,
+  or ``status`` for ``<command>``:
+
+  .. tabbed-parsed-literal::
+    :tabs: "Linux or Mac OS X",Windows
+    :mapping: linux,windows
+    :dependent: linux-windows
+    :languages: console,shell-session
+    
+    .. Linux or Mac OS X
+    
+    $ docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk <command>
+
+    $ docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk start
+    $ docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk restart
+    $ docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk stop
+
+    # To see the status:
+    $ docker exec -it cdap-standalone /opt/cdap/sdk/bin/cdap sdk status
+
+    .. Windows
+    
+    > docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk <command>
+
+    > docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk start
+    > docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk restart
+    > docker exec -d cdap-standalone /opt/cdap/sdk/bin/cdap sdk stop
+
+    # To see the status:
+    > docker exec -it cdap-standalone /opt/cdap/sdk/bin/cdap sdk status
```

Comment:
These are all `docker exec ${opts} ${container_name} cdap sdk ${command}` where:
- ${opts} is one of `-it` or `-d` for foreground/background
- ${container_name} is always `cdap-sdk` (--name from the `docker run`)
- `cdap` script is in the PATH, always... we can specify it, but it's unnecessary
- ${command} is the argument to `cdap sdk`

You can do check `status` of two ways, as you've documented, where you do it in the foreground (-it) and parse status with your eyes, or you can do it in the background like the others and check exit status, which is better for scripted use. I think this is fine for the user-oriented case, though.